hw/net/rocker/rocker_hw.h

```diff
@@ -146,6 +147,7 @@ typedef struct rocker_tlv {
 enum {
     ROCKER_TLV_CMD_UNSPEC,
     ROCKER_TLV_CMD_TYPE,                /* u16 */
+    ROCKER_TLV_CMD_WORLD,               /* u32 - world id */
```

Comment:
Ok, those reasons make sense.  I guess OF-DPA is the exception.  We might be able to promote of-dpa to this model without breaking backward-compat with older drivers by assuming if ROCKER_TLV_CMD_WORLD is not defined in the tlvs then it must be an of-dpa cmd.